nuclear/utils/lockfile.py

```diff
@@ -0,0 +1,12 @@
+LOCKFILE_VERSION = "1.0.0"
+
+def save(user, repo, version, url, sha256):
```

Comment:
Could you add type information here as well to be consistent with the rest of the code? :D